openrtb-core/src/main/protobuf/openrtb-ext.proto

```diff
@@ -243,6 +243,19 @@ message SKAdNetworkRequest {
 message ImpExt {
   // Publisher's SKAdNetwork information.
   optional SKAdNetworkRequest skadn = 1;
+
+  // To differentiate between impression opportunities
+  // that support Protected Audience API on-device auction from
+  // those that only support traditional server-side exchange auction
+  enum AuctionEnvironment {
+    // Traditional server-side auctions
+    SERVER_SIDE_AUCTION = 0;
+    // Requests with Protected Audience API support,
+    // in which a contextual auction runs on the exchange's servers
+    // and the interest group bidding and the final auction runs in the browser
+    ON_DEVICE_INTEREST_GROUP_AUCTION = 1;
+  }
+  optional int32 ae = 2;
```

Comment:
FYI Google is using `[default = 0]`. (Their equivalent is at https://developers.google.com/authorized-buyers/rtb/downloads/openrtb-adx-proto -- note we do not need to use the same index numbers as them in extensions.)